synapse/cortex.py

```diff
@@ -398,6 +416,7 @@ def _runCryoLoop(self):
 
         online = False
         tankurl = self.conf.get('splice:cryotank')
+        tankname = tankurl.split('/')[3]  # FIXME find better way to get tank name
```

Comment:
@vEpiphyte - do you think there is a better way to get the tank name? Should i just store it in conf?

Reply:
do we need the tank name?

Reply:
fixed